tests/generation/test_utils.py

```diff
@@ -691,40 +667,39 @@ def test_model_parallel_beam_search(self):
                 new_model.generate(
                     input_ids,
                     attention_mask=attention_mask,
-                    max_length=max_length,
+                    max_new_tokens=self.max_new_tokens,
```

Comment:
everywhere we call model.generate, if we pas this `max_new_token`, why not in the `setup` update the `model.generation_config.max_new_tokens`?

Reply:
Hmm, you mean the "setup" in each model's test file, so that we already have the "max_new_tokens" linked to model? That means we need to initialize model from config when calling `_get_input_ids_and_config` and then set `model.generation_config.max_new_tokens`. 

I can do it, seems like it will not cause any errors. But I am not sure if that's what you mean